.azure-pipelines.yml

```diff
@@ -474,9 +474,9 @@ jobs:
         artifact: macos_amd64_app
 
 
-  - job: s2_windows_installer_build
-    displayName: Windows installer build
-    condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  - job: s2_windows_installer_content_build
```

Comment:
The azure CI fails with this error because of the renaming:
> Job s4_wheel_pypi_upload depends on unknown job s2_windows_installer_build.

Reply:
done.